src/js/Sort/CellFormatter.js

```diff
@@ -150,10 +157,11 @@ function ftDateTimeToNumber(text) {
 	const time = text.match(/(?:\s|^)(\d{1,2}(?:[.](\d{2}))?)(pm|am)$/);
 	// Get date.
 	const month = date && date[1] ? date[1] : null;
-	// Get full month name from a given month e.g. 'January' for 'Jan'.
-	const matchingMonthNames = monthNames.filter((name) => name.indexOf(month) !== -1);
-	// Get the index of the matching month name.
-	const monthIndex = matchingMonthNames && matchingMonthNames[0] ? monthNames.indexOf(matchingMonthNames[0]) : null;
+	// Get index of the month name from a given month e.g. 'January' for 'Jan'.
+	let monthIndex = month && monthNames.findIndex((name) => name.startsWith(month));
```

Comment:
breaking release alert, IE11 doesn't support `findIndex`, we'd need to require a new polyfill :(

Reply:
I've added a migration guide :-)